### DISCUSSION TITLE
Quickly adjusting the volume for an individual non-default audio device from the system tray

### DISCUSSION THREAD

**Discussion by DeveloperBlue** [File-New-Project/EarTrumpet#693] [Ideas]
When you click the EarTrumpet icon in the system tray, it either launches the volume sliders for the current Default Playback Device, or you can hit the arrow and expand to manage *all* playback devices and their programs at once.

I only need to ever change one specific playback device. It happens that this device is not the default playback device, so to access it I need to expand the volume menu. For me, I get a large menu with too many audio devices and programs- meaning I need to scroll around to find the audio device I'm looking for.

It would be ideal if in EarTrumpet settings, we could have only the volume controls for a specific audio playback device (that's not the default), the same way it appears for the default playback device.

tl;dr Option to display controls for only one non-default audio device the same way you currently display the controls for only the default audio device.
